.env

```diff
@@ -1,4 +1,5 @@
 COMPOSE_PROJECT_NAME=annotations
+COMPOSER_DEV=
```

Comment:
from its name, looks like a boolean argument, but it's options; could be `COMPOSER_ARGS` (similarly for the Dockerfile argument naming)

Reply:
Originally tried to make it a boolean, but `--dev` doesn't exist on `dump-autoload` (it's deprecated on `install`). `args` sound ok, has to apply to both commands though (unless we separate them).